apps/ejabberd/src/ejabberd_app.erl

```diff
@@ -238,3 +240,10 @@ load_drivers([Driver | Rest]) ->
                           [erl_ddll:format_error(Reason)]),
             exit({driver_loading_failed, Driver, Reason})
     end.
+
+init_metrics() ->
```

Comment:
I would move this function to the mongoose_metrics module to be consistent with `mongoose_metrics:remove_all_metrics()`